### PR TITLE
Upgrade - Update code to the new content renderer (dupe)

### PR DESCRIPTION
**Pull request by aspirisen**
See https://github.com/prezly/content-renderer-react-js/pull/19

### CODE REVIEW

**Comment by kudlajz** [comment]
Is this a duplicate of https://github.com/prezly/theme-nextjs-bea/pull/119?

File: components/SlateRenderer/components/ContactCard.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import type { ContactNode } from '@prezly/slate-types';
+import React from 'react';
+
+import { ContactCard as ContactCardComponent } from '@/components';
+
+import styles from '../SlateRenderer.module.scss';
+
+interface ContactCardComponentProps {
```
Comment (e1himself) [inline components/SlateRenderer/components/ContactCard.tsx:8]: I suggest using `Props` naming in such cases. Especially when this interface is not exported publicly. I see the prefix as unnecessary visual noise.

**Comment by aspirisen** [comment]
@e1himself should this PR be closed in favour of yours? 
If yes, could you increase the slate renderer version number

**Comment by e1himself** [comment]
@aspirisen Yeah, I was just thinking about which one it's better to keep. Let me upgrade the package version in #121.

**Comment by aspirisen** [comment]
Good, by the way, why i have failed nextjs builds, maybe i should have access to it? @kudlajz

**Comment by e1himself** [comment]
@aspirisen I'll grant you access. But JFYI I am sure you can reproduce the error locally by running `npm run build`.

**Comment by kudlajz** [comment]
It failed because of
```
Type error: Module '"@prezly/content-renderer-react-js"' has no exported member 'Html'.
```

**Comment by e1himself** [comment]
> I'll grant you access. 

Nope, I can't. Please ask Camil to let you in.

**Comment by kudlajz** [comment]
Let_Me_In

**Pull request closed without merging**